src/App.js

```diff
@@ -1,23 +1,74 @@
-import logo from './logo.svg';
+import contactsArr from './contacts.json'
+import {useState} from 'react'
 import './App.css';
 
+
+
 function App() {
+  let fiveContacts =  [...contactsArr.slice(0, 5)]
+  
+  let [contactList, setContacts] = useState(fiveContacts)
+
+  const addContact = ( () => {
+    let random = Math.floor( (Math.random() * contactsArr.length) -1)
```

Comment:
here you forget the logic for adding contacts once. We have already selected 5 contacts from contactsArr and should avoid selecting these actors again